src/peer/peer_manager.cpp

```diff
@@ -348,7 +348,8 @@ bool PeerManager::InitialSyncCompleted() const {
 void PeerManager::InitialSync() {
     while (!interrupt_) {
         std::this_thread::sleep_for(std::chrono::milliseconds(100));
-        auto now = time(nullptr);
+        auto now         = time(nullptr);
+        static auto next = std::chrono::steady_clock::now();
```

Comment:
Seems no need to set it to be `static` since the var can be defined outside the `while` loop